docs/notications/notifications-shema.yaml

```diff
@@ -0,0 +1,55 @@
+definitions:
+  notifications:
+    type: array
+    items:
+      type: object
+      properties:
+        _id:
+          type: string
+        user:
+          type: string
+          $ref: '#/components/user'
+        userRole:
+          type: string
```

Comment:
pass here enum of roles and to definition below

Reply:
added